### PR TITLE
[T4A2][T10-A4]ZavierOng

### PR DESCRIPTION
**Pull request by Zavier-Ong**

### CODE REVIEW

**Comment by E0002878** [comment]
Ready for review

File: src/seedu/addressbook/data/person/Person.java [hunk 1]
```diff
@@ -16,6 +16,8 @@
     private Address address;
 
     private final UniqueTagList tags;
+    private int sequenceNumber;
+    private static int nextSequenceNumber = 0;
```
Comment (mauris) [inline src/seedu/addressbook/data/person/Person.java:20]: do take note of the semantics here. when we talk about nextSequenceNumber, usually we take this number for the object created. unless this is named `currentSequenceNumber` and set as 0, it should set to 1 instead.

File: src/seedu/addressbook/parser/Parser.java [hunk 2]
```diff
@@ -50,7 +50,7 @@ public Parser() {}
      * @param userInput full user input string
      * @return the command based on the user input
      */
-    public Command parseCommand(String userInput) {
+    public static Command parseCommand(String userInput) {
```
Comment (mauris) [inline src/seedu/addressbook/parser/Parser.java:53]: were there instances of using these methods that need updating?

**Comment by mauris** [comment]
Some comments added. Feel free to comment if you have any questions. Once done, you may the PR after reading comments.

**Pull request closed without merging**